src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,253 +1,186 @@
 package nextstep.subway.line;
 
-import static org.assertj.core.api.Assertions.*;
+import static nextstep.subway.line.LineAcceptanceTestUtils.*;
+import static nextstep.subway.station.StationAcceptanceTestUtils.*;
 
 import java.time.LocalDateTime;
 import java.util.Arrays;
 import java.util.HashMap;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
-import java.util.Set;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
 
-import io.restassured.RestAssured;
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
 import nextstep.subway.AcceptanceTest;
 import nextstep.subway.line.dto.LineResponse;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
 
 @DisplayName("지하철 노선 관련 기능")
 public class LineAcceptanceTest extends AcceptanceTest {
-    @DisplayName("지하철 노선을 생성한다.")
-    @Test
-    void createLine() {
-        // given
+    private Station station1;
+    private Station station2;
+    private Station station3;
+    private Station station4;
+
+    @Override
+    @BeforeEach
+    public void setUp() {
+        super.setUp();
+
+        station1 = 지하철_역_등록되어_있음("강남역");
```

Comment:
```suggestion
        강남역 = 지하철_역_등록되어_있음("강남역");
```

이런식으로 활용해도 가독성 측면으로는 괜찮아 보이네요 :) ㅎㅎ